UnityProject/Assets/Scripts/Objects/Doors/FalseWallTileBlend.cs

```diff
@@ -0,0 +1,245 @@
+using Doors;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Tiles;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+
+namespace Objects.Doors
+{
+    public class FalseWallTileBlend : MonoBehaviour
+    {
+        [HideInInspector] private SpriteHandler SPRITE_HANDLER;
+        [HideInInspector] private RegisterDoor REGISTER_DOOR;
+        [Tooltip("Empty tile to spawn when false wall texture is setted.")]
+        [SerializeField] private BasicTile falseTile = null;
+        [HideInInspector] private TileChangeManager TILE_CHANGE_MANAGER;
+        [HideInInspector] private DoorController DOOR_CONTROLLER;
+        [HideInInspector] private Vector3Int falseWallPosition;
+        [HideInInspector] private List<LayerTile> tilesPresendtedNear = new List<LayerTile>();
+        [Tooltip("Updates tile sprite in update")]
+        [HideInInspector] public bool needUpdate = true;
+        [HideInInspector] private static Dictionary<Vector3Int, FalseWallTileBlend> falseWallPresented = new Dictionary<Vector3Int, FalseWallTileBlend>();
+
+        private Sprite[] sprites;
+
+        private static readonly int[] map =
+        {
+            0, 2, 4, 8, 1, 255, 3, 6, 12, 9, 10, 5, 7, 14, 13, 11, 15, 19, 38, 76, 137, 23, 39, 46, 78, 77, 141, 139, 27, 31, 47, 79, 143, 63, 111, 207, 159,
+            191, 127, 239, 223, 55, 110, 205, 155, 175, 95
+        };
+
+        private void Start()
+        {
+            SPRITE_HANDLER = GetComponentInChildren<SpriteHandler>();
+            REGISTER_DOOR = GetComponentInChildren<RegisterDoor>();
+            TILE_CHANGE_MANAGER = GetComponentInParent<TileChangeManager>();
+            falseWallPosition = REGISTER_DOOR.WorldPositionServer + new Vector3Int(-1, -1);
```

Comment:
can you explain this? Why south?

Reply:
Don`t know why but this world position returns shifted point instead of real